doc-surrealdb_versioned_docs/version-latest/surrealql/statements/define/access/index.mdx

```diff
@@ -67,6 +67,18 @@ DEFINE ACCESS account ON DATABASE TYPE RECORD
 ;
 ```
 
+## With `AUTHENTICATE` clause
+
+The authenticate clause can be used to change the record identifier returned by the `SIGNIN` and `SIGNUP` clauses or replace the identifier provided in the token when authenticating `WITH JWT` in the context of `DEFINE ACCESS ... TYPE RECORD`. 
```

Comment:
You cannot return or replace anything from the `AUTHENTICATE` clause with `WITH JWT`, that only is the case for record access. (@gguillemas please correct me if I'm wrong)

Reply:
Oh yes i missed the title there! Oops!